bin/alias_schemas

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+import click
+import json 
+import pathlib
+import shutil
+
+
+@click.command()
+@click.argument(
+    'aliases-path',
+    type=click.Path(
+        dir_okay=False,
+        file_okay=True,
+        writable=False,
+        exists=True,
+    ),
+    required=True
+)
+@click.argument(
+    'base-dir',
+    type=click.Path(
+        dir_okay=True,
+        file_okay=False,
+        writable=True,
+        exists=True,
+    ),
+    required=True
+)
+def main(aliases_path, base_dir):
+    base_dir = pathlib.Path(base_dir)
+
+    with open(aliases_path) as f:
+        aliases_parsed = json.load(f)
+
+    for dest_ns, doctypes in aliases_parsed.items():
+        for dest_dt, alias_info in doctypes.items():
+            source_ns = alias_info['src-namespace']
+            source_dt = alias_info['src-doctype']
```

Comment:
The naming is all over the place, I'd consider making the names consistent. For example:

```
src-namespace -> source-namespace
source_ns -> source_namespace
```